### PR TITLE
feat(dashboard): aggregate and time series charts per score

### PR DESCRIPTION
**Pull request by marliessophie**

### CODE REVIEW

File: web/src/features/dashboard/components/ScoresTable.tsx [hunk 1]
```diff
@@ -131,18 +189,23 @@ export const ScoresTable = ({
           <RightAlignedCell key="one">1</RightAlignedCell>,
         ]}
         rows={data.map((item, i) => [
-          item.scoreName,
+          <LeftAlignedCell
+            key={`${i}-name`}
+          >{`${getScoreDataTypeIcon(item.scoreDataType)} ${item.scoreName} (${item.scoreSource.toLowerCase()})`}</LeftAlignedCell>,
```
Comment (marliessophie) [inline web/src/features/dashboard/components/ScoresTable.tsx:194]: Opted to keep score label cohesive across charts/tables for now, can look into improving UI feel of the score source display in the future for charts (and potentially across platform), but felt more cohesive this way

File: web/src/features/dashboard/components/LeftAlignedCell.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { cn } from "@/src/utils/tailwind";
+import { type ReactNode } from "react";
+
+export const LeftAlignedCell = ({
```
Comment (marcklingen) [inline web/src/features/dashboard/components/LeftAlignedCell.tsx:4]: i'd inline this, a simple text-left is easier to understand on a div than in a component in another file (that's why we use tailwind)

File: web/src/features/dashboard/components/ScoresTable.tsx (hunk 1 above)
Comment (marcklingen) [reply web/src/features/dashboard/components/ScoresTable.tsx:194]: looks good to me!

File: web/src/features/dashboard/components/LeftAlignedCell.tsx (hunk 2 above)
Comment (marcklingen) [reply web/src/features/dashboard/components/LeftAlignedCell.tsx:4]: just saw the right aligned cell, i'd also remove it as this makes our code more convoluted

File: web/src/features/dashboard/components/hooks.ts [hunk 3]
```diff
@@ -46,17 +46,35 @@ type ChartData = {
...
 
+type FieldMappingItem = {
+  uniqueIdentifierColumns: {
+    accessor: string;
+    formatFct?: (value: string) => string;
+  }[];
+  valueColumn: string;
+};
+
+function getLabelValue(
```
Comment (marcklingen) [inline web/src/features/dashboard/components/hooks.ts:57]: can you add a breif comment on what this does?

**Review by marcklingen — COMMENTED** [review]
overall looks good to me, 2 small remarks

File: web/src/features/dashboard/components/hooks.ts (hunk 3 above)
Comment (marliessophie) [reply web/src/features/dashboard/components/hooks.ts:57]: Change name of function

**Pull request merged**